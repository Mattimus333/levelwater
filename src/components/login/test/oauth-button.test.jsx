import React from 'react';
import { shallow } from 'enzyme';
import OauthButton from '../oauth-button.jsx';
import toJson from 'enzyme-to-json';

describe('oauth button', () => {
  it('renders a snapshot', () => {
    const tree = shallow(<OauthButton />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
