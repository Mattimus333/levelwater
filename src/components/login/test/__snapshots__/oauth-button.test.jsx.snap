exports[`oauth button renders a snapshot 1`] = `
<div
  className="o-auth-button">
  <a
    href="http://levelwater-server.herokuapp.com/auth/google">
    <img
      alt="Login With Google"
      src="google-oauth-button.png" />
  </a>
</div>
`;
